src/reducer/cartReducer.jsx

```diff
@@ -0,0 +1,17 @@
+const cartReducer = (state, action) => {
+  switch (action.type) {
+    case "ADD_TO_CART":
+      return { ...state, cartList: [...action.payload] };
+
+    case "REMOVE_FROM_CART":
+      return { ...state, cartList: [...action.payload] };
+
+    case "UPDATE_CART_QUANTITY":
+      return { ...state, cartList: [...action.payload] };
```

Comment:
suggestion - the return value for these reducer are same, so you can replace it with a single reducer, for example "SET_CART"

Reply:
Right! I'll make the changes